chemprop/features/featurization.py

```diff
@@ -503,12 +507,14 @@ class BatchMolGraph:
     * :code:`max_num_bonds`: The maximum number of bonds neighboring an atom in this batch.
     * :code:`b2b`: (Optional) A mapping from a bond index to incoming bond indices.
     * :code:`a2a`: (Optional): A mapping from an atom index to neighboring atom indices.
+    * :code:`b2br`: (Optional): A mapping from f_bonds to real bonds in molecule recorded in targets.
```

Comment:
What is a f_bond?

Reply:
A mapping from a bond index to a list of bond features.
https://github.com/chemprop/chemprop/blob/master/chemprop/features/featurization.py#L295